### PR TITLE
Devcontainer features native installation support

### PR DESCRIPTION
**Pull request by mml5bg**
# Description
This PR implements officially supported approach for devcontainer features installation.

## Azure DevOps PBI/Task reference

AB#_[PBI/Task number]_

## Checklist

* [x] Vehicle App can be started with dapr run and is connecting to vehicle data broker
* [ ] Vehicle App can process MQTT messages and call the seat service
* [ ] Vehicle App can be deployed to local K3D and is running
* [ ] Created/updated tests, if necessary. Code Coverage percentage on new code shall be >= 70%.
* [ ] Extended the documentation in Velocitas repo
* [ ] Extended the documentation in README.md

* [x] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [x] Devcontainer can be re-built successfully

* [ ] Release workflow is passing

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -27,11 +27,12 @@
 			}
 		}
 	},
+	"features": {
```
Comment (doosuu) [inline .devcontainer/devcontainer.json:30]: Do we really need this? It is already installed into the base container.

**Review by doosuu — CHANGES_REQUESTED** [review]

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (mml5bg) [reply .devcontainer/devcontainer.json]: Good observation, it is not needed anymore, I have deleted it.

**Review by dennismeister93 — APPROVED** [review]
LGTM! :)

**Review by doosuu — APPROVED** [review]
LGTM

**Pull request merged**